### PR TITLE
Update PHPUnit documentation link

### PR DESCRIPTION
**Pull request by martixy**

### CODE REVIEW

File: testing.rst [hunk 1]
```diff
@@ -974,7 +974,7 @@ Learn more
 * :doc:`/components/css_selector`
 
 .. _`PHPUnit`: https://phpunit.de/
-.. _`documentation`: https://phpunit.de/manual/current/en/
+.. _`documentation`: https://phpunit.de/documentation.html
```
Comment (xabbuh) [inline testing.rst]: Should we rather use https://phpunit.readthedocs.io/ instead?

Comment (xabbuh) [reply testing.rst]: IMO that's inconsistent with what we do elsewhere. We always link to the English version of docs we link to. For most users that saves one click and if you are interested in reading a localised version, it's easy to switch the language.

Comment (martixy) [reply testing.rst]: Either's fine. This doesn't warrant that much of an extended discussion.

**Comment by javiereguiluz** [comment]
@martixy thanks a lot for fixing this wrong link!

**Pull request closed without merging**